lib/web_ui/dev/felt

```diff
@@ -53,6 +53,14 @@ install_deps() {
   (cd "$FLUTTER_DIR/web_sdk/web_engine_tester"; $PUB_PATH get)
 }
 
+MAC_OS=`system_profiler SPSoftwareDataType | grep Mac`
```

Comment:
Is `system_profiler` available on Linux? My computer doesn't have it.

Reply:
Maybe use `uname`?

Reply:
thanks so much for the suggestion